build/dotnet/install-old.ps1

```diff
@@ -0,0 +1,108 @@
+#
```

Comment:
The two old scripts are the result of `git mv install.ps1 install-old.ps1`. Will get removed once we've updated